src/localization/bundles/en.json

```diff
@@ -712,7 +712,7 @@
     "server.deploy.progress.delete_network_agents.failure": "Error: Failed to delete network agents for compute host {0}. {1}",
     "server.deploy.installos.failure": "Failed to install SLES OS on compute host {0}.",
     "server.deploy.retry.confirm": "Retry will rerun the failed playbook. Check the log for errors and if there were any issues, resolve them before proceeding. \n\n Do you want to continue?",
-    "server.deploy.failure.cancel.confirm": "Check the log for errors and if there were any issues, resolve them before proceeding. \n\n Do you want to continue?",
+    "server.deploy.failure.cancel.confirm": "Cancelling will close the window, and any error messages will no longer be available to view.\n\nDo you want to cancel?",
```

Comment:
cancel confirm doesn't apply to your screen anymore.

Reply:
I know this does not apply any more, but the old message was incorrectly worded, making it appear the they were "proceeding" and "continuing" instead of cancelling. Going back to the summary page is certainly "closing the window", because we have a button there that says "close" under normal circumstances.

Reply:
@GarySmith  that is fine with me.